scene/2d/camera_2d.cpp

```diff
@@ -60,9 +60,11 @@ void Camera2D::_update_scroll() {
 
 		Size2 screen_size = _get_camera_screen_size();
 		Point2 screen_offset = (anchor_mode == ANCHOR_MODE_DRAG_CENTER ? (screen_size * 0.5) : Point2());
+		Point2 adj_screen_pos = camera_screen_center - (screen_size * 0.5);
 
-		get_tree()->call_group(group_name, "_camera_moved", xform, screen_offset);
-	};
+		// TODO: Remove xform and screen_offset when ParallaxBackground/ParallaxLayer is removed
+		get_tree()->call_group(group_name, "_camera_moved", xform, screen_offset, adj_screen_pos);
```

Comment:
```suggestion
		get_tree()->call_group(group_name, SNAME("_camera_moved"), xform, screen_offset, adj_screen_pos);
```

Reply:
Good catch! As long as we're modifying camera_2d.cpp, might as well clean it up.